scalafmt-tests/src/test/resources/scala3/OptionalBraces_fold.stat

```diff
@@ -1324,3 +1324,22 @@ val bar = foo({
      x += 1
   arr
 })
+<<< lambda
```

Comment:
could you please add this test to the 4th file, `Optional Braces.stat`?

Reply:
Done!